diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/StringTemplateFormatRule.kt

```diff
@@ -77,4 +80,8 @@ class StringTemplateFormatRule(private val configRules: List<RulesConfig>) : Rul
             }
         }
     }
+
+    private fun bracesCanBeOmitted(node: ASTNode) = !node.hasAnyChildOfTypes(CALL_EXPRESSION)
+            && !node.treeNext.text.startsWith("_")
```

Comment:
I think we shouldn't remove braces if there are any symbols other than spaces, not just an underscore. Or is it handled somewhere else?